config/routes.rb

```diff
@@ -33,9 +33,13 @@
       put 'update_cart'
       delete 'empty_cart'
     end
-    resources :equipment_objects
+    resources :equipment_items
   end
 
+  get 'equipment_objects' => redirect('equipment_items')
+  get 'equipment_objects/:command' =>
```

Comment:
This doesn't actually work, I don't think `subdomain` is what you want. Check [here](http://guides.rubyonrails.org/routing.html#redirection) and our [generalized matcher](https://github.com/YaleSTC/reservations/blob/1081_rename_equip_obj_to_equipment_item/config/routes.rb#L161).